Description: remove checking erlang version
 Without this patch, the configure will try to run erlang to simply check if
 the version is supported by ejabberd. Instead, we do this test statically.
Author: Johan Oudinet <johan.oudinet@gmail.com>

Index: ejabberd/configure.ac
===================================================================
--- ejabberd.orig/configure.ac	2014-10-20 11:04:14.751256865 +0200
+++ ejabberd/configure.ac	2014-10-20 11:03:47.983578623 +0200
@@ -37,10 +37,8 @@ AC_ARG_ENABLE(erlang-version-check,
 	[Check Erlang/OTP version @<:@default=yes@:>@])])
 case "$enable_erlang_version_check" in
 	yes|'')
-		ERLANG_VERSION_CHECK([$REQUIRE_ERLANG_MIN],[$REQUIRE_ERLANG_MAX])
 		;;
 	no)
-		ERLANG_VERSION_CHECK([$REQUIRE_ERLANG_MIN],[$REQUIRE_ERLANG_MAX],[warn])
 		;;
 esac
 
